src/mlagility/cli/run_slurm.sh

```diff
@@ -1,19 +1,23 @@
 #!/usr/bin/env bash
 script="$1"
 args="$2"
+target_file_folder="$3"
 
-# Add any data caches used by third party model hubs (huggingface, torch hub, etc.)
+# shellcheck disable=SC1090
+source /net/home/"$USER"/miniconda3/etc/profile.d/conda.sh
+conda activate tracker_slurm
+export USING_SLURM="TRUE"
 export HF_DATASETS_CACHE="/net/ml-cache/huggingface"
 export TRANSFORMERS_CACHE="/net/ml-cache/huggingface"
 export HF_HOME="/net/ml-cache/huggingface"
 export XDG_CACHE_HOME="/net/ml-cache/huggingface"
 export TORCH_HOME="/net/ml-cache/torch-hub"
-
-# shellcheck source=/dev/null
-source /net/home/"$USER"/miniconda3/etc/profile.d/conda.sh
-conda activate groqit_slurm
-export USING_SLURM="TRUE"
-# shellcheck disable=SC2046
-# shellcheck disable=SC2116
+export TORCH_HUB="/net/ml-cache/torch-hub"
```

Comment:
These kind of exports are pretty spicy for an open source project. Most users wont have this directory. Maybe these exports just need to go into the .bashrc files of Groq developers?

Reply:
Agreed. I'm only deduplicating the infrastructure here. Added that as a todo of https://github.com/groq/mlagility/issues/89